docs/src/notebooks/7_community_model.jl

```diff
@@ -0,0 +1,125 @@
+# # Building and analysing a small community model
+
+#md # [![](https://mybinder.org/badge_logo.svg)](@__BINDER_ROOT_URL__/notebooks/@__NAME__.ipynb)
+#md # [![](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/notebooks/@__NAME__.ipynb)
+
+# Here we will use `COBREXA` to build and analyze a small community model  
+# consisting of three *E. coli* mutants using the `CoreModel`. We will use an
+# objective function that enforces equal growth rates.
+
+# ## Load the base model
+
+!isfile("e_coli_core.json") &&
+    download("http://bigg.ucsd.edu/static/models/e_coli_core.json", "e_coli_core.json");
+
+using COBREXA
+using Tulip
+
+# ## Load the models and inspect fba solutions
+
+base_model = load_model(CoreModel, "e_coli_core.json")
+
+cytbd_knockout = remove_reactions(base_model, "CYTBD")
+sol = flux_balance_analysis_dict(cytbd_knockout, Tulip.Optimizer)
+println("Cytochrome oxidase knockout μ = ", sol["BIOMASS_Ecoli_core_w_GAM"])
```

Comment:
println not needed, it's a notebook so you can just let it print itself in one line.

Also, I'd add some coment about what the results actually mean (although it can be seen from the code, the new users will not be able to read it this way)

Reply:
extra suggestion, perhaps rename to eno_knockout_model (very hungarian but helps comprehesion)